resources/wikia/Resources.php

```diff
@@ -183,6 +184,13 @@
 		],
 	],
 
+	'moment' => [
```

Comment:
@Grunny Ah, I see. Removing it from amd.shared and instead loading it in onBeforePageDisplay inside the extension (using $wgOut->addModules("moment");) will accomplish what I want, right?